server/src/meguca/assets/common.go

```diff
@@ -1,8 +1,29 @@
 package assets
 
 import (
-	"meguca/util"
+	"io/ioutil"
+	"path/filepath"
 	"sync"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/go-playground/log"
+
+	"meguca/common"
+	"meguca/util"
+)
+
+// Export to avoid circular dependency
+func init() {
+	common.GetVideoNames = GetVideoNames
```

Comment:
Why do we need this instead of just importing `assets` from `templates`?

Reply:
Circular dependencies, need to be able to recompile templates from assets module.

Reply:
Oh, Okay.